frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/DivergedFromRemoteDialog.kt

```diff
@@ -63,42 +62,43 @@ class DivergedFromRemoteDialog(
           getString(
             "action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.do-not-resolve-option"
           ),
-          DivergeResolutionOption.DO_NOT_SYNC
+          DivergenceResolutionOption.DO_NOT_SYNC
         ).commentCompat(getString("action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.do-not-resolve-option.comment"))
       }
-
-      rowCompat {
-        radioButton(
-          getString(
-            "action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.force-push-option"
-          ),
-          DivergeResolutionOption.FORCE_PUSH
-        ).commentCompat(getString("action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.force-push.comment"))
-      }
-
-      if (withResetOption) {
-        rowCompat {
+      when (relationToRemote) {
+        SyncToRemoteStatus.DivergedFromAndOlderThanRemote -> rowCompat {
```

Comment:
Nit: newline before `rowCompat` for consistency with the case below (unless forced this way by spotless)